test/testdata/fixtures/single-dc/k8ssandra.yaml

```diff
@@ -61,11 +61,19 @@ spec:
           - name: "cassandra"
           - name: "server-system-logger"
           - name: busybox
-            image: busybox
+            image: busybox:stable
             command: ["sleep", "3600"]
             volumeMounts:
               - name: busybox-vol
                 mountPath: /var/lib/busybox
+            lifecycle:
+              preStop:
+                exec:
+                  command:
+                    - sh
+                    - '-c'
+                    - sleep 180
```

Comment:
What is the graceperiod in this test? I think by default that's 180s, so wouldn't we end up with forced deletion?

Reply:
> What is the graceperiod in this test? I think by default that's 180s, so wouldn't we end up with forced deletion?

I think that's ok, since all we want is to delay the deletion of the pod in order to delay the deletion of the cassdc.